contracts/PRBMath.sol

```diff
@@ -131,197 +131,220 @@ library PRBMath {
+                }
+                if (x & 0x10000000000000 > 0) {
+                    result = (result * 0x1000B175EFFDC76BA) >> 64;
+                }
+                if (x & 0x8000000000000 > 0) {
+                    result = (result * 0x100058BA01FB9F96D) >> 64;
+                }
+                if (x & 0x4000000000000 > 0) {
+                    result = (result * 0x10002C5CC37DA9492) >> 64;
+                }
+                if (x & 0x2000000000000 > 0) {
+                    result = (result * 0x1000162E525EE0547) >> 64;
+                }
+                if (x & 0x1000000000000 > 0) {
+                    result = (result * 0x10000B17255775C04) >> 64;
+                }
+            }
+
+            if (x & 0xFF0000000000 > 0) {
+                if (x & 0x800000000000 > 0) {
+                    result = (result * 0x1000058B91B5BC9AE) >> 64;
+                }
+                if (x & 0x400000000000 > 0) {
+                    result = (result * 0x100002C5C89D5EC6D) >> 64;
+                }
+                if (x & 0x200000000000 > 0) {
+                    result = (result * 0x10000162E43F4F831) >> 64;
+                }
+                if (x & 0x100000000000 > 0) {
+                    result = (result * 0x100000B1721BCFC9A) >> 64;
+                }
+                if (x & 0x80000000000 > 0) {
+                    result = (result * 0x10000058B90CF1E6E) >> 64;
+                }
+                if (x & 0x40000000000 > 0) {
+                    result = (result * 0x1000002C5C863B73F) >> 64;
+                }
+                if (x & 0x20000000000 > 0) {
+                    result = (result * 0x100000162E430E5A2) >> 64;
+                }
+                if (x & 0x10000000000 > 0) {
+                    result = (result * 0x1000000B172183551) >> 64;
+                }
+            }
+
+            if (x & 0xFF00000000 > 0) {
+                if (x & 0x8000000000 > 0) {
+                    result = (result * 0x100000058B90C0B49) >> 64;
+                }
+                if (x & 0x4000000000 > 0) {
+                    result = (result * 0x10000002C5C8601CC) >> 64;
+                }
+                if (x & 0x2000000000 > 0) {
+                    result = (result * 0x1000000162E42FFF0) >> 64;
+                }
+                if (x & 0x1000000000 > 0) {
+                    result = (result * 0x10000000B17217FBB) >> 64;
+                }
+                if (x & 0x800000000 > 0) {
+                    result = (result * 0x1000000058B90BFCE) >> 64;
+                }
+                if (x & 0x400000000 > 0) {
+                    result = (result * 0x100000002C5C85FE3) >> 64;
+                }
+                if (x & 0x200000000 > 0) {
+                    result = (result * 0x10000000162E42FF1) >> 64;
+                }
+                if (x & 0x100000000 > 0) {
+                    result = (result * 0x100000000B17217F8) >> 64;
+                }
+            }
+
+            if (x & 0xFF00000000 > 0) {
```

Comment:
Here is bug, value should be `0xFF000000` instead of `0xFF00000000`

Reply:
yup, @andreivladbrg fixed this via #179.